src/main/java/org/commcare/formplayer/objects/SerializableFormSession.java

```diff
@@ -1,248 +1,155 @@
 package org.commcare.formplayer.objects;
 
+import lombok.Getter;
+import lombok.Setter;
 import org.hibernate.annotations.GenericGenerator;
 import org.springframework.data.annotation.CreatedDate;
 import org.springframework.data.jpa.domain.support.AuditingEntityListener;
 
 import javax.persistence.*;
 import java.io.Serializable;
 import java.time.Instant;
+import java.util.Date;
 import java.util.Map;
 
-/**
- * Created by willpride on 1/19/16.
- */
 @Entity
-@Table(name = "formplayer_sessions")
+@Table(name="formplayer_sessions")
 @EntityListeners(AuditingEntityListener.class)
 public class SerializableFormSession implements Serializable{
+    @Getter
     @Id
-    @GeneratedValue( generator = "uuid" )
-    @GenericGenerator(name = "uuid", strategy = "org.hibernate.id.UUIDGenerator")
+    @GeneratedValue( generator="uuid" )
+    @GenericGenerator(name="uuid", strategy="org.hibernate.id.UUIDGenerator")
     private String id;
 
+    @Getter
     @Version
     private Integer version;
 
-    @Column(name="instancexml")
-    private String instanceXml;
-
-    @Column(name="formxml")
-    private String formXml;
-
-    private String username;
-
-    @Column(name="initlang")
-    private String initLang;
-
-    @Column(name="sequenceid")
-    @Convert(converter = IntStringConverter.class)
-    private Integer sequenceId;
-
-    @Column(name="sessiondata")
-    @Convert(converter = ByteArrayConverter.class)
-    private Map<String, String> sessionData;
-
-    private String domain;
-
-    @Column(name="posturl")
-    private String postUrl;
-
-    @Column(name="menu_session_id")
-    private String menuSessionId;
-    private String title;
-
-    // legacy field to be removed once `dateCreated` is fully populated
-    @Column(name="dateopened")
+    /**
+     * Deprecated: to be removed once `dateCreated` is fully populated
```

Comment:
@snopoke Would we be able to deprecate `dateCreated` after a week from when this PR was deployed (Given we only keep incomplete forms for a week)  ? Just wondering if I should add pagination to [deprecated methods here](https://github.com/dimagi/formplayer/blob/master/src/main/java/org/commcare/formplayer/repo/FormSessionRepo.java#L32) as well for Incomplete forms pagination support.

Reply:
I don't think it's worth adding pagination to the deprecated methods. The plan is to remove these within the next week or so.

Reply:
Awesome. Thanks for confirming.